satellite/contact/endpoint.go

```diff
@@ -84,16 +91,19 @@ func (endpoint *Endpoint) CheckIn(ctx context.Context, req *pb.CheckInRequest) (
 	}, nil
 }
 
-func (endpoint *Endpoint) pingBack(ctx context.Context, req *pb.CheckInRequest, peerID storj.NodeID) (bool, string, error) {
+func (endpoint *Endpoint) pingBack(ctx context.Context, req *pb.CheckInRequest, peerID storj.NodeID) (_ bool, _ string, err error) {
+	defer mon.Task()(&ctx)(&err)
+
 	client, err := newClient(ctx, endpoint.service.dialer, req.Address, peerID)
 	if err != nil {
+		endpoint.log.Info("pingBack dialing error", zap.String("nodeID", peerID.String()), zap.String("error", err.Error()))
```

Comment:
we'll want a field for `req.Address` here

Reply:
hmm. should we change it to ping the IP instead fo the addr?

Reply:
I'm thinking we should do that, yeah. Like change GetNetwork() into GetIPAndNetwork() and then pass that into pingBack for the address, both so we can log the IP that we got and so we don't have to resolve everything twice (since we do too much dns querying already).

But it could be done in a separate PR if you don't feel like it is related enough here.

Reply:
i looked into this a little and seems more involved so i will put in a different PR.